Plots/XY/NCL_xy_18.py

```diff
@@ -0,0 +1,124 @@
+"""
+NCL_xy_18.py
+============
+Concepts illustrated:
+
+- Filling the area between two curves in an XY plot
+- Labeling the bottom X axis with years
+- Drawing a main title on three separate lines
+- Calculating a weighted average
+- Changing the size/shape of an XY plot using viewport resources
+- Manually creating a legend
+- Overlaying XY plots on each other
+- Maximizing plots after they've been created
+
+See the [original NCL example](https://www.ncl.ucar.edu/Applications/Scripts/xy_18.ncl)
+"""
+
+###############################################################################
+# Basic imports
+import numpy as np
+import xarray as xr
+from matplotlib import pyplot as plt
+import matplotlib.ticker as tic
+
+###############################################################################
+# Open files and read in monthly data
```

Comment:
should add some text about this workaround and why its necessary. I can help...

Reply:
Done.  I've added some text explaining more of what is being done in this example.